### ISSUE TITLE
Ironic issue with Naming/MethodName and Style/CommentedKeyword

### ISSUE THREAD

**Issue by forthrin** [rubocop/rubocop#14673]
## Steps to reproduce the problem

```ruby
def LEIA(bar)
  puts bar
end

# Use snake_case for method names. [Naming/MethodName]

def LUKE(bar) # rucocop:disable Naming/MethodName
  puts bar
end

# Naming/MethodName: Use snake_case for method names.
# Style/CommentedKeyword: Do not place comments on the same line as the def keyword.

def SOLO(bar) # rucocop:disable Naming/MethodName, Style/CommentedKeyword
  puts bar
end

# Naming/MethodName: Use snake_case for method names.
# Style/CommentedKeyword: Do not place comments on the same line as the def keyword.

```

## Expected behavior

Book LEIA, but not LUKE and SOLO.

## Actual behavior

One could say the snake bites it own tail.

## RuboCop version

```
ruby 3.4.7 (2025-10-08 revision 7a5688e2a2) +PRISM [arm64-darwin24]
rubocop   2025-11-21 f0983bde0
```

**Comment by lovro-bikic** [comment]
There seems to be a typo in the disable directive: `rucocop:disable`.

```patch
-def LUKE(bar) # rucocop:disable Naming/MethodName
+def LUKE(bar) # rubocop:disable Naming/MethodName
   puts bar
 end


-def SOLO(bar) # rucocop:disable Naming/MethodName, Style/CommentedKeyword
+def SOLO(bar) # rubocop:disable Naming/MethodName, Style/CommentedKeyword
   puts bar
 end
```

After fixing the typo, `Naming/MethodName` offenses are not registered for `LUKE` and `SOLO`. `SOLO` also registers a `Lint/RedundantCopDisableDirective` offense because there's no `Style/CommentedKeyword` offense.

**Comment by forthrin** [comment]
😳 The embarrassment. 😖 The humiliation.
